pkg/controller/masteruserrecord/sync_test.go

```diff
@@ -86,14 +86,11 @@ func setupSynchronizerItems() (toolchainv1alpha1.MasterUserRecord, toolchainv1al
 	base := toolchainv1alpha1.UserAccountSpecBase{
 		NSLimit: "limit",
 		NSTemplateSet: toolchainv1alpha1.NSTemplateSetSpec{
-			TierName: "basic",
-			ClusterResources: &toolchainv1alpha1.NSTemplateSetClusterResources{
-				Revision: "654321a",
-			},
+			TierName:         "basic",
+			ClusterResources: &toolchainv1alpha1.NSTemplateSetClusterResources{},
```

Comment:
why empty? The `ClusterResources` should be either `nil` or it should contain some `TemplateRef` set.
```suggestion
			ClusterResources: &toolchainv1alpha1.NSTemplateSetClusterResources{
				TemplateRef: "basic-clusterresources-654321a",
			},
```

Reply:
fixed here: d98f4bc8c02bfd5fb7180b49b944aa70a5dc37b2